tools/pythonpkg/tests/fast/test_multithread.py

```diff
@@ -19,40 +20,42 @@ def connect_duck(duckdb_conn):
     assert out == [(42,), (84,), (None,), (128,)]
 
 
+def everything_succeeded(results: List[bool]):
+    return all([result == True for result in results])
```

Comment:
```suggestion
    return all(results)
```

Reply:
I know, and expected this comment, but I prefer to be explicit